packages/syft/src/syft/service/settings/settings.py

```diff
@@ -265,6 +269,22 @@ class ServerSettings(SyftObject):
     pwd_token_config: PwdTokenResetConfig = PwdTokenResetConfig()
     allow_guest_sessions: bool = True
 
+    @field_validator("organization")
+    def organization_length(cls, v: str) -> str:
+        if len(v) < MIN_ORG_NAME_LENGTH:
+            raise ValueError(
+                f"'organization' must be at least {MIN_ORG_NAME_LENGTH} characters long"
+            )
+        return v
+
+    @field_validator("name")
+    def name_length(cls, v: str) -> str:
+        if len(v) < MIN_SERVER_NAME_LENGTH:
+            raise ValueError(
+                f'"name" must be at least {MIN_SERVER_NAME_LENGTH} characters long'
+            )
+        return v
+
```

Comment:
pydantic has [`constr`](https://docs.pydantic.dev/2.0/usage/types/string_types/#arguments-to-constr) which supports validation for min and max str length, e.g `name: constr(min_length = MIN_SERVER_NAME_LENGTH)` which I think would be cleaner here, but needs to check for autosplat and serialization.